HHS/Head-Start-TTADP
# [PROD] Add percentage to dashboard, fix dashboard title spacing, Goals List

## 513

### How to test

- The percentage of recipients served an AR now shows on the regional dashboard
- The dashboard title's top margin now matches the AR landing page and the recipient search

### Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-513

## 456

Needed to add two missing scopes to goals, region and recipient id. Added some test utilities to create/destroy goals to aid in testing. Check pointing at the backend so someone else can pickup the ticket and do the frontend as I won't have time to work on the frontend for a few days.

### How to test

Nothing to test, only change is on the backend.

### Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-456

## 632

Certain filter components expect an array from the query, but it wasn't being decoded as such, and the JS app was crashing

### How to test

Create a page with specialist role filters, copy and paste the URL into another tab and confirm the filter is still applied as expected

### Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-632

## 453

Add Goal List to Goals and Objectives tab. This is currently behind a feature flag. 

### How to test

With the feature flag enabled view a recipients from the search results then click the 'Goals and Objectives' tab. You should see a list of all goals for that recipient for that region. 

NOTE: Objectives expander will be added as part of TTAHUB-568.

### Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-453

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
